src/files-and-videos/videos/VideoThumbnail.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { VideoFile } from '@edx/paragon/icons';
+import {
+  Badge,
+  Button,
+  Icon,
+  Image,
+} from '@edx/paragon';
+import FileInput, { useFileInput } from '../FileInput';
+import messages from './messages';
+
+const VideoThumbnail = ({
+  thumbnail,
+  displayName,
+  id,
+  imageSize,
+  handleAddThumbnail,
+  videoImageSettings,
+  status,
+  // injected
+  intl,
+}) => {
+  const fileInputControl = useFileInput({
+    onAddFile: (file) => handleAddThumbnail(file, id),
+    setSelectedRows: () => {},
+    setAddOpen: () => false,
+  });
+  const allowThumbnailUpload = videoImageSettings?.videoImageUploadEnabled;
+
+  let addThumbnailMessage = 'Add thumbnail';
+  if (allowThumbnailUpload) {
+    if (thumbnail) {
+      addThumbnailMessage = 'Edit thumbnail';
+    }
+  }
+  const supportedFiles = videoImageSettings?.supportedFileFormats
+    ? Object.values(videoImageSettings.supportedFileFormats) : null;
+  let isUploaded = false;
+  switch (status) {
+  case 'Ready':
```

Comment:
can you add indentation here?

Reply:
linter throws an error when case statements are indented